src/components/cart-page/CartProductCard.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+
+function CartProductCard({ product }) {
+  const [quantity, setQuantity] = useState(1);
+
+  return (
+    <div className="bg-regal-blue-dark rounded shadow-lg h-min">
+      <div className="h-full  flex flex-col">
+        <div className="badge-container">
+          {product.newArrival ? (
+            <span className="absolute font-bold card-badge badge bg-rose-500 text-white">
+              New
+            </span>
+          ) : (
+            ""
+          )}
+
```

Comment:
Suggestion : you can use product.newArrival && (.....) as you need to maintain one condition so that you can avoid using "" <---- this , in the ternary operator. && will do the job!! :)

Reply:
okay, i will implement this